### DISCUSSION TITLE
filter

### DISCUSSION THREAD

**Discussion by shinebrillant** [enrique-lozano/Monekin#258] [General]
There are filters in the dashboard
There are filters in transactions
There is filtering in more/statistics

Can it be merged into the same UI interface? Apply filtering on the dashboard, and more /statistics should also apply this filtering.

![Screenshot_20241208-230834](https://github.com/user-attachments/assets/b3cf2e26-2dde-4bd5-98ef-726bcfec3a21)

![Screenshot_20241208-230843](https://github.com/user-attachments/assets/4dc93321-af91-4d69-98bb-de00c00f7390)

![Screenshot_20241208-230858](https://github.com/user-attachments/assets/6cae4f45-98bb-4c63-978a-343c1e469884)

The time range has an "always", I don't quite understand what this range refers to, I also need a time range of "since records began".

![Screenshot_20241208-230838](https://github.com/user-attachments/assets/16bb41e6-997d-4bab-8975-2cf490ff71ac)

**Comment by enrique-lozano** [comment]
Hello, thanks for the comment. This is not in the plans for the near future. The stats and the dashboard page has the time range selector. In the first one is has sense cause you can navigate to previous/next periods and you can apply the rest of the filters too.

In the second one, the dashboard, it would have more sense, but honestly the dashboard page it's only for a quick look of your current situation. If you want a more detailed view with a lot of more options and filters you have the stats page. 99% of the competition do not allow any filtering in the dashboard and I think this is the correct approach to not overload the UI.

Still looking for a way to better unify the UI, but I see no problem in here.

The other thing that you mentioned, "always" means some or less what you look for, it is something like "since the beginning of times"
